src/commands/cont.rs

```diff
@@ -0,0 +1,35 @@
+use crate::api;
+use crate::constants;
+use crate::models;
+use api::ApiClient;
+use chrono::Utc;
+use colored::Colorize;
+use models::{ResultWithDefaultError, TimeEntry};
+
+pub struct ContinueCommand;
+
+impl ContinueCommand {
+    pub async fn execute(api_client: impl ApiClient) -> ResultWithDefaultError<()> {
+        let time_entries = api_client.get_time_entries().await?;
+        match time_entries.first() {
+            None => println!("{}", "No time entries in last 90 days".red()),
+            Some(time_entry) => {
+                let start = Utc::now();
+                let time_entry_to_create = TimeEntry {
+                    start,
+                    stop: None,
+                    duration: -start.timestamp(),
+                    created_with: Some(constants::CLIENT_NAME.to_string()),
+                    ..time_entry.clone()
+                };
```

Comment:
Can we keep the helpers for starting and stop time-entries?

@tcrammond is working on the start command for #8 which could benefit from this common implementation and keeping things dry ☔

Reply:
Sure thing! We can move these to a `common` crate or something

Reply:
That works 🚀